### PR TITLE
Included a short grid generation tutorial 

### PR DESCRIPTION
**Pull request by gabyneg**
This is regarding issue #21, I have updated the README.md under tools/grid/ providing a little more info on the two grid tools available.

### CODE REVIEW

**Comment by yichengt900** [comment]
@gabyneg, thanks for opening this PR. It looks great to me. I will keep it open for a while so others have a chance to review it.

File: tools/grid/README.md [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+  - Interpolate data between different grids. 
+  - Adjust grid resolutions. 
+    
+### Generate grids using ESMG 
+- **Set up conda environment**:  
+    - clone the repository from GitHub: 
+```bash 
+git clone https://github.com/ESMG/gridtools.git 
+``` 
+  - **Create a conda environment**: When creating the conda environemnt, issues might arise from incopatibility between environemnts. [This tutorial](https://github.com/ESMG/gridtools/blob/de0a18c1ce0807748aa70023300dfc415277bd4c/docs/conda/README.md?plain=1#L70) was very helpful providing alternative istallation methods. Here I use alternative installation method (2):
```
Comment (yichengt900) [inline tools/grid/README.md:27]: incompatibility?

Comment (yichengt900) [inline tools/grid/README.md:27]: providing "installation" methods.

**Comment by yichengt900** [comment]
Hi @gabyneg , thanks again for this contribution! I only have two tiny comments for this PR. I will approve it once you address all of them.  Feel free to let me know if you have any questions.

File: tools/grid/README.md (hunk 1 above)
Comment (gabyneg) [reply tools/grid/README.md:27]: Here, I meant "incompatibility between packages." From my understanding, the GridTools Conda environment hasn't been consistently maintained, which can cause issues when installing the environment. However, the original tutorial provides alternative methods to help with the installation (see the section [here](https://github.com/ESMG/gridtools/blob/de0a18c1ce0807748aa70023300dfc415277bd4c/docs/conda/README.md?plain=1#L70)). I used method (2) and still had to manually install a few packages to make it work. Let me know if this explanation makes sense or if it is still too confusing.

Comment (yichengt900) [reply tools/grid/README.md:27]: @gabyneg, thanks for your clarification. It is my bad that I did not make it clear in my original comment. Is 'incopatibility' a typo? If so, it should be 'incompatibility.

Comment (yichengt900) [reply tools/grid/README.md:27]: Same here. "installation" should be "installation".

Comment (yichengt900) [reply tools/grid/README.md:27]: Same here. "istallation" should be "installation".

Comment (gabyneg) [reply tools/grid/README.md:27]: Yes, I will fix that! thank you!

**Review by yichengt900 — APPROVED** [review]
Thanks @gabyneg! Changes look good, Approve!

**Pull request merged**